.github/workflows/serverless-integration.yml

```diff
@@ -0,0 +1,39 @@
+name: "Serverless Integration Tests"
+
+on:
+  push:
+    paths:
+      - 'cmd/serverless/**'
+      - 'pkg/serverless/**'
+      - 'test/integration/serverless/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout datadog-agent repository
+        uses: actions/checkout@v2
+        with:
+          path: go/src/github.com/DataDog/datadog-agent
+
+      - name: Set up Node 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      
+      - name: Install Serverless Framework
+        run: sudo yarn global add serverless --prefix /usr/local
+
+      - name: Checkout the datadog-lambda-extension repository
+        uses: actions/checkout@v2
+        with:
+          repository: DataDog/datadog-lambda-extension
+          path: go/src/github.com/DataDog/datadog-lambda-extension
+
+      - name: Run tests
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
do we want to indicate that those secrets are serverless-related by adding a `SERVERLESS` prefix to those secrets?

Reply:
Excellent idea!